lua/neogit/buffers/editor/init.lua

```diff
@@ -77,26 +77,27 @@ function M:open(kind)
         return pad(mapping[name] and mapping[name][1] or "<NOP>", padding)
       end
 
-      -- stylua: ignore
-      local help_lines = {
-        "#",
-        "# Commands:",
-        string.format("#   %s Close", pad_mapping("Close")),
-        string.format("#   %s Submit", pad_mapping("Submit")),
-        string.format("#   %s Abort", pad_mapping("Abort")),
-        string.format("#   %s Previous Message", pad_mapping("PrevMessage")),
-        string.format("#   %s Next Message", pad_mapping("NextMessage")),
-        string.format("#   %s Reset Message", pad_mapping("ResetMessage")),
-      }
-
-      help_lines = util.filter_map(help_lines, function(line)
-        if not line:match("<NOP>") then -- mapping will be <NOP> if user unbinds key
-          return line
-        end
-      end)
+      if not config.values.disable_editor_help then
+        local comment_char = git.config.get_comment_char()
+        -- stylua: ignore
+        local help_lines = {
+          string.format("%s", comment_char),
+          string.format("%s Commands:", comment_char),
+          string.format("%s   %s Close", comment_char, pad_mapping("Close")),
+          string.format("%s   %s Submit", comment_char, pad_mapping("Submit")),
+          string.format("%s   %s Abort", comment_char, pad_mapping("Abort")),
```

Comment:
I think the `Previous`, `Next`, and `Reset` commands got dropped on accident.

Reply:
Not dropped by me. I just changed what was there.

Reply:
Ah, there were more changes in editor. Will  fix that as well.

Reply:
All good, I merged it very recently, probably some time between you forking and opening a PR :)